### DISCUSSION TITLE
FUSE meeting notes: Aug 1 2023

### DISCUSSION THREAD

**Discussion by orso82** [ProjectTorreyPines/FUSE.jl#384] [General]
Orso
* hackathon last week
    * Worked with David on issues for rebaselining FPP design
    * Worked with Brendan on time dependence
* Time dependence
    * Started coding and getting close to something
    * Brian: Heater time stepping in TRANSP, Astra algorithm in a IPP note

David
* Lot of good progress in hackathon
* Running on OMEGA
    * Issue with distributed optimizations in version 1.9x
    * Issue with Jupyter Lab in version 1.8.x
    * Will discuss with JuliaHub folks soon

Discussion: Savanah River
* Integration into FUSE
* Process to get non-citizens vetted takes months
* Necessarily a citizen
* Text file for inputs, executable, text outputs
* Tom volunteering, perhaps a good synergy with Jackson's BOP modeling?
* possible target of JuliaSim surrogate modeling?

Jackson
* Presented ThermalSystemsModel.jl package at hackathon
* Working to get steady state models into FUSE

Kevin
* PR to get max (vs total) leakage through the blanket
* Minor terminology changes
* Deterministic neutron solver
    * Can generate 2D geometry, but meshing hangs
    * 1D can make the geometry and mesh it, and run
    * 1D had some convergence issues
* Getting Amber up-to-date
    * Learning at least the paramak build

Brendan
* Optimization of TEQUILA
* Re-thinking fitting of flux surfaces in TEQUILA to make it more robust
* Discussed time dependence in FUSE

Discussion: Tokamak energy
* Visited by Tim
* Using TGLF in ASTRA and having difficulties converging
* Issue with community developed codes, no support, also TRANSP
* Cannot use ITER tools
* 4-5 people
* Michele Romanelli contact
* Interested in running OMFIT/STEP

Discussion: comparison with ASTRA modeling of AUG
* T. Luda paper
* how do we compare?
* FUSE SOL is 0D. Can we upgrade to 1D?  Question for Jerome & INFUSE

Tom
* Setup Nan to validate TGLFNN in FUSE
* Found some bug
    * Number of thermal species
    * Timeout of TGLF simulations with large number of species
* Training FPP+DIII-D database for SAT1 (SAT0 is available)

Nan
* Thoughts about modeling of impurities in FUSE

Adriana
* Amani put Adriana in touch with Leo for financial modeling
* Leo wants to understand what we are doing
    * Prepared presentation
    * Bring him up to speed with what we are thinking
* Organizing notes for INFUSE collaboration about costing buildings
    * Who to partner with?
        * Mostly with fission background
    * What would be SOW?
    * What would be the deliverable?
    * A bite-size project, not multi-institutional, hopefully not sub-contractors
    * Next round call in December, with proposals in early February
    * Should ask Raffi to join, and Leo in his technical role

Tim
* SOFE
    * Talks are online
    * What the big players are doing
        * CHINA: huge progam, both BEST (2027) and CFTR being built (2030)
        * UKAEA
        * US is behind
        * CFS is small scale compared to others
    * Interesting of PROCESS using multi-objective optimization
        * Started with genetic algorithms 
        * Looking at bayesian methods --> probability --> Risk
* JuliaCon
    * Computing oriented
    * Many interesting packages
        * Impressed by Gene framework
        * JuliaSim great for surrogate models

**Comment by jguterl** [comment]
Discussion: comparison with ASTRA modeling of AUG

   - T. Luda paper
   - how do we compare?
   - FUSE SOL is 0D. Can we upgrade to 1D? Question for Jerome & INFUSE

Please add references (poster/paper) here.
